Xext/security.c

```diff
@@ -1015,14 +1016,14 @@ SecurityExtensionInit(void)
     ret &= AddCallback(&ClientStateCallback, SecurityClientState, NULL);
     ret &= AddCallback(&ExtensionAccessCallback, SecurityExtension, NULL);
     ret &= AddCallback(&ExtensionDispatchCallback, SecurityExtension, NULL);
+    ret &= AddCallback(&ServerAccessCallback, SecurityExtension, NULL);
```

Comment:
Is it correct that the second argument here is `SecurityExtension`, but in `DeleteCallback` it’s `SecurityServer`?

Reply:
you're right. it needs to be SecurityServer.
thanks for spotting this